tests/unit/algos/list/delimiterMaster.spec.js

```diff
@@ -296,7 +514,7 @@ function getListingKey(key, vFormat) {
                 /* When filtered, it should return FILTER_SKIP and set the prvKey. It
                  * should not be added to the result content or common prefixes. */
                 assert.strictEqual(delimiter.filter(obj), FILTER_SKIP);
-                assert.strictEqual(delimiter.NextMarker, undefined);
+                // assert.strictEqual(delimiter.NextMarker, undefined);
```

Comment:
I am unsure why having the NextContinuationToken be set to undefined is desirable or needed here and on line 533. Both cases occur after filtering a deleteMarker key.